ajax/libs/jsSocials/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "jsSocials",
+  "filename": "dist/jssocials.min.js",
```

Comment:
It will be useful to us to change `dist/jssocials.min.js` to `jssocials.min.js`.
Because the `filename` need to point to the main file of a lib, and our `basePath` of auto-update is `dist`, the `basePath` will be ignored when copying over to the CDN.
Thank you.